src/stories/Library/event-description/event-description.scss

```diff
@@ -0,0 +1,50 @@
+.event-description {
+  @include container(medium);
+  padding: $s-md;
+  @include breakpoint-s {
```

Comment:
jeg forstår godt du vil bruge grid areas her, men: 

Fx. hvis jeg tilføjer flere sted
I listen listDescription, hvis man her tilføjer enten flere **steder** eller **pris** som jeg tænker vil være tilfældet, vil dit "grid-area: Links" blive skubbet længere og længere ned. 

Jeg tænker at vi nok senere skal have en fold in/ud funktionalitet på i det mindste **sted** hvis nu der er mange. Men når den så bliver foldet ud vil den også blive skubbet. 

Jeg synes det skal laves om til, at ListDescriptions størrelse, ikke skubber til det, som lige nu, er dit "grid-area:links"

Reply:
Jeg kan godt se din pointe med, at det i fremtiden kunne give problemer. Jeg valgte denne løsning for at undgå en wrapper div, men jeg har tilføjet den nu.